expr.go

```diff
@@ -132,6 +132,86 @@ func (neq NotEq) ToSql() (sql string, args []interface{}, err error) {
 	return Eq(neq).toSql(true)
 }
 
+// Lt is syntactic sugar for use with Where/Having/Set methods.
+// Ex:
+//     .Where(Lt{"id": 1})
+type Lt map[string]interface{}
+
+func (lt Lt) toSql(opposite, orEq bool) (sql string, args []interface{}, err error) {
+	var (
+		exprs   []string
+		opr     string = "<"
+		nullOpr string = "IS"
```

Comment:
Understandable, any suggestions?

It could throw an error like is the case for an array or slice. What do you think?